app/main/views/frameworks.py

```diff
@@ -103,6 +103,43 @@ def framework_services(framework_slug):
     ), 200
 
 
+@main.route('/frameworks/digital-outcomes-and-specialists/declaration/<string:section_id>',
```

Comment:
This will want to work from the existing route `'/frameworks/<string:framework_slug>/declaration/<string:section_id>` - I appreciate this is intended for prototyping, but is there a danger that too much will go in here that won't easily generalise to that route?  There doesn't look to be anything here that couldn't work in that general route - is there some other reason for hard-coding the framework in here?